draft-iab-rfcv3-preptool-latest.mkd

```diff
@@ -402,7 +405,9 @@ None.
 Security Considerations {#securitycons}
 =====
 
-None.
+Steps in this document attempt to prevent the &lt;artwork&gt; and &lt;sourcecode&gt; entities from exposing
+the contents of files outside the directory in which the document being processed
+resides.
```

Comment:
We don't have any steps in the document to prevent x:include from exposing the contents of files outside the directory, so no.